restAPI/restApi.go

```diff
@@ -196,16 +196,17 @@ func (restApi *restAPIImpl) getInstallations(c *gin.Context) (*InstallationsResp
 // @Description  This installs an extension
 // @Id           installExtension
 // @Produce      json
-// @Success      200 {object} interface{}
+// @Success      200 {object} string
 // @Param        dbHost query string true "Hostname of the Exasol DB to manage"
```

Comment:
should we use lowercase or capitalization consistently?

Reply:
✅